hikari/impl/interaction_server.py

```diff
@@ -45,8 +45,7 @@
 
     import aiohttp.typedefs
 
-    # This is only imported here for internal type-hints and otherwise needs
-    # to be inline imported as it's an optional dependency.
+    # This is kept inline as pynacl is an optional dependency.
     from nacl import signing
```

Comment:
Wouldnt this make typechecking fail if they dont have the dependency installed?

Reply:
Wdym make type checking fail? Also it's only used internally